_static/css/course.css

```diff
@@ -0,0 +1,127 @@
+.admonition, .topic {
```

Comment:
Does this `course.css` file have some connection to the CSS in the a-plus-rst-tools? Or why do we want to have custom styles here in the aplus-manual course? 

Was this file already in the manual before your changes? I have forgotten about this. Maybe there was an idea that some good styles could be added or updated in the a-plus-rst-tools theme. It would good to have good default styles in the platform so that courses do not always define similar styles inside the course repository. Of course, courses can still define custom CSS if they want to have custom, unique styles, but if all courses copy the same CSS, then it is not necessary to define the CSS in the course repos.

Reply:
The file was already in the course. I just wanted to move it to the css folder and add some "small fixes" to it, but actually, I implemented these "small fixes" directly in the theme apluslms/a-plus-rst-tools/pull/88. So the question here is, do we want to apply the style fixes to the course itself or make them more general by applying the changes to the aplus theme?. 

I personally think that we can apply the changes proposed in apluslms/a-plus-rst-tools/pull/88 and I will simply use this PR to move the CSS files to the css folder and update the conf.py file with more recent start and end dates as well as applying changes related to the `use_wide_column` and version of the manual (hopefully, we will version the manual at some point).

Let me know what you think about it. @Mankro

Reply:
Let's add the general style improvements to a-plus-rst-tools in the PR https://github.com/apluslms/a-plus-rst-tools/pull/88. Does the course.css file in this course-templates repo have any meaningful styles then? If not, let's remove it.

Reply:
The `course.css` file has some styles that modified the styled-topic directive. I think we can keep those changes here, but we should think after the Mathew's auditory if we want to change the `aplus-theme` in the https://github.com/apluslms/a-plus-rst-tools/tree/master/theme/aplus and include those changes in that repository.